### PR TITLE
Show subscribe block support doc on help center on editor when url param `help-center=subscribe-block` exists.

### PR DESCRIPTION
**Pull request by yansern**
Related to #84594.

P2s:
- pdDOJh-2JD-p2
- pdDOJh-2Ee-p2#comment-2017

## Proposed Changes

* Show subscribe block support doc on help center on editor when url param `help-center=subscribe-block` exists.
* Related PR: https://github.com/Automattic/jetpack/pull/34329

## Testing Instructions

* Go to `apps/editing-toolkit`.
* Run `yarn dev --sync`.
* Sandbox your site.
* Go to `your-site.wordpress.com/wp-admin/site-editor.php?canvas=edit&help-center=subscribe-block`.
* You should see the help center show up automatically with the subscribe block support doc page loaded.

## Screenshot

image

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by TimBroddin — APPROVED** [review]
Nice job!

File: apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js [hunk 1]
```diff
@@ -45,6 +46,20 @@ function HelpCenterContent() {
...
+			actionHooks.forEach( ( actionHook ) => {
+				if ( actionHook.condition() ) {
+					actionHook.action();
+				}
+			} );
+		}, 0 );
+		return () => clearTimeout( timeout );
+		// Only want to run this once
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [] );
```
Comment (alshakero) [inline apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js]: I would move this effect inside the `useActionHooks`.

**Review by alshakero — DISMISSED** [review]
Great! One note.

File: apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js (hunk 1 above)
Comment (yansern) [reply apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js]: How would that look like? What would useActionHooks return?

Comment (alshakero) [reply apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js]: It doesn’t have to return anything. Calling it is enough IMHO.

Comment (yansern) [reply apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js]: Okay gotcha.

**Pull request merged**